backend/selenium_tests/grievance/grievance_tickets/test_grievance_tickets.py

```diff
@@ -204,3 +348,112 @@ def test_grievance_tickets_create_new_ticket_referral(
         assert "New" in pageGrievanceDetailsPage.getTicketStatus().text
         assert "Not set" in pageGrievanceDetailsPage.getTicketPriority().text
         assert "Not set" in pageGrievanceDetailsPage.getTicketUrgency().text
+
+    def test_grievance_tickets_needs_adjudication(
+        self,
+        add_grievance_needs_adjudication: None,
+        pageGrievanceTickets: GrievanceTickets,
+        pageGrievanceDetailsPage: GrievanceDetailsPage,
+        pageIndividuals: Individuals,
+    ) -> None:
+        pageGrievanceTickets.getNavGrievance().click()
+        assert "Grievance Tickets" in pageGrievanceTickets.getGrievanceTitle().text
+        pageGrievanceTickets.getTabSystemGenerated().click()
+        pageGrievanceTickets.getTicketListRow()[0].click()
+        pageGrievanceDetailsPage.getSelectAllCheckbox().click()
+        pageGrievanceDetailsPage.getPersonIcon()
+        assert "person-icon" in [
+            ii.get_attribute("data-cy")
```

Comment:
consider something more verbose than `ii`

Reply:
I will update it in separate PR later today or tomorrow morning